devlab/jenkins/prep_filter.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+tenant_name="tenant1"
+
+filter="configs/filter.yaml"
+cfg="devlab/config.ini"
+
+src=$(grep grizzly_ip $cfg | awk '{print $3}')
+ssh_user=$(grep src_ssh_user $cfg | awk '{print $3}')
+ssh_cmd="ssh $ssh_user@${src}"
+
+if [[ $1 == "--tenant" ]]; then
+a="awk '{print \$2}'"
```

Comment:
Is it here just to follow the 79 symbols per line rule?.. It's quite unreadable, also shell scripts don't have to follow pep8, I think we can have 120 symbols for non-python code.